internal/clients/diffreviewer/github/github_service.go

```diff
@@ -203,7 +198,7 @@ func (s *Service) LoadConfig(nightfallConfigFileName string) (*nightfallconfig.C
 	if nightfallConfig.MaxNumberRoutines < nightfall.MaxConcurrentRoutinesCap {
 		maxNumberRoutines = nightfallConfig.MaxNumberRoutines
 	} else {
-		maxNumberRoutines = nightfall.MaxConcurrentRoutinesCap
+		maxNumberRoutines = nightfallconfig.DefaultMaxNumberRoutines
```

Comment:
Should we also just handle this logic inside of `nightfallconfig.GetNightfallConfigFile`, since this is also the same for all clients?